test/unit/nedap/utils/test/api.cljc

```diff
@@ -202,10 +218,13 @@
                "#{:to-change :from :to}"
                `(sut/expect () :unexpected () :signature 4 :keys)
 
-               "0 should be different from 0"
+               "0 should not match 0"
                `(sut/expect () :to-change 0 :from 0 :to 0)
 
-               "^#:unit.nedap.utils.test.api{:wat true} {} should be different from ^#:unit.nedap.utils.test.api{:wat true} {}"
+               "0 should not match 0"
+               `(sut/expect () :with ~'= :to-change 0 :from 0 :to 0)
```

Comment:
maybe :with should be the last option to be passed, so that the invocation reads like English

i.e. expect should be "close" to :to-change

Can this be `assert`ed?

Reply:
Changing the order seems like a good suggestion, albeit asserting seems too much as maps have no order, and this yields no functional difference

Reply:
> Changing the order seems like a good suggestion, albeit asserting seems too invasive as maps have no order, and this yields no functional difference

At the same time it can be observed that this API was designed to read like English. Linting or code-reviewing that later is costlier

...Ultimately the fact that kwargs are a map can be considered an impl detail, since one can opt to destructure `:with ~'= :to-change 0 :from 0 :to 0`  as either `& opts` (vector) or `& {:as opts}` (map)